documentation/get-started/domain-map.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: sub-navigation
+order: 30
+title: Domain map
+description: Set of statements about how domain entities relate to each other.
+---
+
+## Direct relationships
+- A case is for a single case type
+- A case type is owned by a single business unit
+- A business unit can own multiple case types
+- A case consists of multiple stages
+- A user can belong to multiple user groups
+- A user group is assigned to a single case type
```

Comment:
I think user groups are assigned to business units, not case types (I am interpreting this from the fact that being in a DCU team gives you read access to all 3 DCU case types). Either that or user groups can be assigned to multiple case types.

Reply:
OK, this is great - I can definitely have not understood correctly!